### ISSUE TITLE
Instagram Reel leads to "Requested content is not available

### ISSUE THREAD

**Issue by thebeautyofcoding** [yt-dlp/yt-dlp#4750]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I remove or skip any mandatory\* field

### Checklist

- [X] I'm asking a question and **not** reporting a bug or requesting a feature
- [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
- [X] I've verified that I'm running yt-dlp version **2022.08.19** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar questions **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Please make sure the question is worded well enough to be understood

I am trying to run yt-dlp as subprocess in Python 3.10 in Windows 10. 
Via the command `p= subprocess.run(["yt-dlp",url,"-J"],stdout=subprocess.PIPE, text=True)` where `url=https://www.instagram.com/reel/ChRtyT4jZTe/` I am able to download (by omitting "-J") or get the download URL ("cdninstagram") of certain Instagram Reels. 
However, half of them fails such as "ChRtT4jZTey" and I get the error: `ChRtyT4jZTe: Requested content is not available, rate-limit reached or login required. Use --cookies, --cookies-from-browser, --username and --password, or --netrc (instagram) to provide account credentials
CompletedProcess(args=['yt-dlp',  '-J', 'https://www.instagram.com/reel/ChRtyT4jZTe/?utm_source=ig_web_copy_link'], returncode=1, stdout='null\n')`
In Bash and cmd in Windows 10, I get the same error, but in PowerShell it works flawless. 
Output in PowerShell:
`{"id": "ChRtyT4jZTe", "formats": [{"url": "https://instagram.fclj3-1.fna.fbcdn.net/v/t50.2886-16/299597967_841195450202344_2542793318561483375_n.mp4?_nc_ht=instagram.fclj3-1.fna.fbcdn.net&_nc_cat=104&_nc_ohc=bFYwwMtK06gAX-UF6Tm&edm=AP_V10EBAAAA&ccb=7-5&oe=63080BA7&oh=00_AT-VBr2MP5pIWGc3UbVo4TdkCJpONdY7quI28T1MRk_ZtA&_nc_sid=4f375e", "width": 750, "height": 1333, "protocol": "https", "ext": "mp4", "video_ext": "mp4", "audio_ext": "none", "format_id": "0", "format": "0 - 750x1333", "resolution": "750x1333", "dynamic_range": "SDR", "http_headers": {"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.77 Safari/537.36", "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8", "Accept-Language": "en-us,en;q=0.5", "Sec-Fetch-Mode": "navigate", 
"Referer": "https://www.instagram.com/"}}], "title": "Video by joeel56", "description": "Monday hustle mode on \ud83d\udc69\ud83c\udffb\u200d\ud83d\udcbb\n Love the new LEDs strips on my wall \ud83e\udd70 trying to get a lot of work done this week. My current sprint is a little bit on delay \ud83c\udf1a how is your week? Btw did u saw my back goes \ud83e\udd97\ud83e\udd9f\ud83e\udd97 \ud83d\ude02\ud83d\ude02 \n.\n.\n.\n\n#webdeveloper #uiux #minimalist#interface #design#uxdesign#setupinspiration #programmerworld #web #userinterface #programmer #application#javascript#setup#code #minimal #website #developer #frontenddeveloper", "duration": 14.536, "timestamp": 1660559595, "uploader_id": "288495926", "uploader": "joeel56 Webdeveloper", "channel": "joeel56", "like_count": 2027, "comment_count": 29, "comments": [{"author": "aniakubow", "author_id": "292805263", "id": "17897864222648806", "text": "I love the LED lighting on your wall\ud83d\ude0d\ud83d\ude0d", "timestamp": 1660575194}, {"author": "alonsovallve", "author_id": "12036590578", "id": "17933159168378883", "text": "Bien \ud83d\ude4c\ud83d\ude4c\ud83d\ude4c", "timestamp": 1660592169}, {"author": "_edgeworks", "author_id": "30807455295", "id": "17922806894414967", "text": "\ud83e\udd97\ud83e\udd9f\ud83d\ude02 #setupIs \ud83d\udd25\ud83d\udd25", "timestamp": 1660600512}, {"author": "who.is.lyss_", "author_id": "1791382322", "id": "17955062998896842", "text": "Absolutely love your setup!!\ud83e\udd29", "timestamp": 1660687356}, {"author": "vanessa_flair_alexander.fikx", "author_id": "51637501075", "id": "18065043523316513", "text": "Hey \ud83d\udc4b\ud83c\udffb We just sent you 
a DM for a potential collaboration. Please reply ASAP to start. Thanks! \ud83d\ude42", "timestamp": 1660727878}, {"author": "vanessa_flair_alexander.fikx", "author_id": "51637501075", "id": "17946907283164915", "text": "Hey \ud83d\udc4b\ud83c\udffb We just sent you a DM for a potential collaboration. Please reply ASAP to start. Thanks! \ud83d\ude42", "timestamp": 1660727914}, {"author": "techpuni", "author_id": "54323309382", "id": "17905431374534705", "text": "That led looks so cool!", "timestamp": 1660733107}, {"author": "superintelligence_", "author_id": "30410161185", "id": "17922202127427708", "text": "Love the new lights\ud83d\ude0d", "timestamp": 1660740085}, {"author": "iblock.om", "author_id": "14785507343", "id": "18313525576061553", "text": "How much cost components computer", "timestamp": 1660797680}], "thumbnails": [{"url": "https://instagram.fclj3-1.fna.fbcdn.net/v/t51.2885-15/299656300_768056184345908_2305085916977115048_n.jpg?stp=dst-jpg_e15_p640x640&_nc_ht=instagram.fclj3-1.fna.fbcdn.net&_nc_cat=108&_nc_ohc=1wPSoUlYW3sAX8pBkvX&edm=AP_V10EBAAAA&ccb=7-5&oh=00_AT8ngqlB-qjppbNvtiwWssb3V6uD3mds-RSW5fGkqIb8Sg&oe=63082106&_nc_sid=4f375e", "width": 640, "height": 1137, "id": "0", "resolution": "640x1137"}, {"url": "https://instagram.fclj3-1.fna.fbcdn.net/v/t51.2885-15/299656300_768056184345908_2305085916977115048_n.jpg?stp=dst-jpg_e15&_nc_ht=instagram.fclj3-1.fna.fbcdn.net&_nc_cat=108&_nc_ohc=1wPSoUlYW3sAX8pBkvX&edm=AP_V10EBAAAA&ccb=7-5&oh=00_AT_-lsVTUk1W6RbBx3M9vmIf5a0HfOxCHdRSFbg27rvDvQ&oe=63082106&_nc_sid=4f375e", "width": 750, "height": 1333, "id": "1", "resolution": "750x1333"}, {"url": "https://instagram.fclj3-1.fna.fbcdn.net/v/t51.2885-15/299656300_768056184345908_2305085916977115048_n.jpg?stp=dst-jpg_e15&_nc_ht=instagram.fclj3-1.fna.fbcdn.net&_nc_cat=108&_nc_ohc=1wPSoUlYW3sAX8pBkvX&edm=AP_V10EBAAAA&ccb=7-5&oh=00_AT_-lsVTUk1W6RbBx3M9vmIf5a0HfOxCHdRSFbg27rvDvQ&oe=63082106&_nc_sid=4f375e", "width": 1080, "height": 1920, "id": "2", "resolution": "1080x1920"}], "http_headers": {"User-Agent": "Mozilla/5.0 (Windows NT 10.0; 
Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.77 Safari/537.36", "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8", "Accept-Language": "en-us,en;q=0.5", "Sec-Fetch-Mode": "navigate", "Referer": "https://www.instagram.com/"}, "webpage_url": "https://www.instagram.com/reel/ChRtyT4jZTe/", "original_url": "https://www.instagram.com/reel/ChRtyT4jZTe/", "webpage_url_basename": "ChRtyT4jZTe", "webpage_url_domain": "instagram.com", "extractor": "Instagram", "extractor_key": "Instagram", "playlist": null, "playlist_index": null, "thumbnail": "https://instagram.fclj3-1.fna.fbcdn.net/v/t51.2885-15/299656300_768056184345908_2305085916977115048_n.jpg?stp=dst-jpg_e15&_nc_ht=instagram.fclj3-1.fna.fbcdn.net&_nc_cat=108&_nc_ohc=1wPSoUlYW3sAX8pBkvX&edm=AP_V10EBAAAA&ccb=7-5&oh=00_AT_-lsVTUk1W6RbBx3M9vmIf5a0HfOxCHdRSFbg27rvDvQ&oe=63082106&_nc_sid=4f375e", "display_id": "ChRtyT4jZTe", "fulltitle": "Video by joeel56", "duration_string": "14", "upload_date": "20220815", "requested_subtitles": null, "_has_drm": null, "requested_downloads": [{"http_headers": {"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.77 Safari/537.36", "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8", "Accept-Language": "en-us,en;q=0.5", "Sec-Fetch-Mode": "navigate", "Referer": "https://www.instagram.com/"}, "url": "https://instagram.fclj3-1.fna.fbcdn.net/v/t50.2886-16/299597967_841195450202344_2542793318561483375_n.mp4?_nc_ht=instagram.fclj3-1.fna.fbcdn.net&_nc_cat=104&_nc_ohc=bFYwwMtK06gAX-UF6Tm&edm=AP_V10EBAAAA&ccb=7-5&oe=63080BA7&oh=00_AT-VBr2MP5pIWGc3UbVo4TdkCJpONdY7quI28T1MRk_ZtA&_nc_sid=4f375e", "width": 750, "height": 1333, "protocol": "https", "ext": "mp4", "video_ext": "mp4", "audio_ext": "none", "format_id": "0", "format": "0 - 750x1333", "resolution": "750x1333", "dynamic_range": "SDR", "epoch": 1661341990, "_filename": "Video by joeel56 [ChRtyT4jZTe].mp4", "__write_download_archive": false}], "url": "https://instagram.fclj3-1.fna.fbcdn.net/v/t50.2886-16/299597967_841195450202344_2542793318561483375_n.mp4?_nc_ht=instagram.fclj3-1.fna.fbcdn.net&_nc_cat=104&_nc_ohc=bFYwwMtK06gAX-UF6Tm&edm=AP_V10EBAAAA&ccb=7-5&oe=63080BA7&oh=00_AT-VBr2MP5pIWGc3UbVo4TdkCJpONdY7quI28T1MRk_ZtA&_nc_sid=4f375e", "width": 750, "height": 1333, "protocol": "https", "ext": "mp4", "video_ext": "mp4", "audio_ext": "none", "format_id": "0", "format": "0 - 750x1333", "resolution": "750x1333", "dynamic_range": "SDR", "epoch": 1661341990, "_type": "video"}`
Somebody knows whether I am doing something wrong or it's a bug? Thanks a lot!

### Provide verbose output that clearly demonstrates the problem

- [ ] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['-vU']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8 
[debug] yt-dlp version 2022.08.19 [48c88e0] (win32_exe) 
[debug] Python 3.8.10 (CPython 64bit) - Windows-10-10.0.19044-SP0  
[debug] Checking exe version: ffmpeg -bsfs 
[debug] Checking exe version: ffprobe -bsfs 
[debug] exe versions: ffmpeg N-104465-g08a501946f-20211104 (setts), ffprobe N-104465-g08a501946f-20211104 
[debug] Optional libraries: Cryptodome-3.15.0, brotli-1.0.9, certifi-2022.06.15, mutagen-1.45.1, sqlite3-2.6.0, websockets-10.3 
[debug] Proxy map: {} 
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: 2022.08.19, Current version: 2022.08.19 
yt-dlp is up to date (2022.08.19)
```

**Comment by bashonly** [comment]
please post verbose logs by adding `-v` to the actual commands you are running, e.g.:
`yt-dlp -v [url]`
`subprocess.run(["yt-dlp",url,"-J","-v"]`

the json output itself is not useful to diagnose the problem

**Comment by thebeautyofcoding** [comment]
Thanks for your quick reply. 
With the `-v`-flag I am getting the following output now:
`[Instagram] ChRtyT4jZTe: Requested content is not available, rate-limit reached or login 
required. Use --cookies, --cookies-from-browser, --username and --password, or --netrc (instagram) to provide account credentials
  File "yt_dlp\extractor\common.py", line 666, in extract
  File "yt_dlp\extractor\instagram.py", line 427, in _real_extract
  File "yt_dlp\extractor\common.py", line 1130, in raise_login_required`

**Comment by bashonly** [comment]
(that is not a complete verbose log)

but if you are now receiving that error message with powershell, we can conclude you are probably being rate-limited by instagram. non-authenticated users are only allowed to view a handful of posts before every attempt is redirected to the login page for the next ~24 hours. if you can't load the post in an incognito/private browsing tab, then yt-dlp won't be able to download it (unless you want to pass cookies or login credentials.)

**Comment by thebeautyofcoding** [comment]
Hi bashonly,
Despite the fact that I get this error with cmd, bash and subprocess for always the same individual reels that I try to download, PowerShell always works. So I can assume, Instagram hasn’t blocked me or so?
As an example, downloading this Reel `https://www.instagram.com/reel/ChRtyT4jZTe/` would never work with bash and cmd, but would always work with PowerShell. There are also other reels that I am never able to download with subprocess, bash and cmd… but then with PowerShell, I am able to.
I also tried to use proxies, but getting same output.

I hope this here is helpful:
`WARNING: [Instagram] ChRtyT4jZTe: No csrf token set by Instagram API 
WARNING: [Instagram] ChRtyT4jZTe: General metadata extraction failed (some metadata might be missing).
WARNING: [Instagram] unable to extract shared data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
WARNING: [Instagram] Main webpage is locked behind the login page. Retrying with embed webpage 
WARNING: [Instagram] unable to extract additional data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version using  yt-dlp -U
ERROR: [Instagram] ChRtyT4jZTe: Requested content is not available, rate-limit reached or login 
required. Use --cookies, --cookies-from-browser, --username and --password, or --netrc (instagram) to provide account credentials
p! CompletedProcess(args=['yt-dlp', '-J', 'https://www.instagram.com/reel/ChRtyT4jZTe/?utm_source=ig_web_copy_link'], returncode=1, stdout='null\n')`

**Comment by bashonly** [comment]
if powershell always works please post a successful verbose log for comparison

**Comment by thebeautyofcoding** [comment]
> if powershell always works please post a successful verbose log for comparison

`[debug] Command-line config: ['https://www.instagram.com/reel/ChRtyT4jZTe/', '-J', '-v']
[debug] Encodings: locale cp1252, fs utf-8, pref cp1252, out utf-8, error utf-8, screen utf-8 
[debug] yt-dlp version 2022.08.14 [5593720] (pip)
[debug] Python 3.10.0 (CPython 64bit) - Windows-10-10.0.19044-SP0  
[debug] Checking exe version: ffmpeg -bsfs
[debug] Checking exe version: ffprobe -bsfs 
[debug] exe versions: ffmpeg N-104465-g08a501946f-20211104 (setts), ffprobe N-104465-g08a501946f-20211104 
[debug] Optional libraries: Cryptodome-3.15.0, brotli-1.0.9, certifi-2021.10.08, mutagen-1.45.1, sqlite3-2.6.0, websockets-10.1
[debug] Proxy map: {} 
[debug] [Instagram] Extracting URL: https://www.instagram.com/reel/ChRtyT4jZTe/ 
[Instagram] ChRtyT4jZTe: Downloading JSON metadata 
[Instagram] ChRtyT4jZTe: Downloading video info 
[Instagram] ChRtyT4jZTe: Downloading embed webpage 
WARNING: [Instagram] unable to extract additional data; please report this issue on  https://github.com/yt-dlp/yt-dlp/issues?q= , filling out the appropriate issue template. Confirm you are on the latest version 
using  yt-dlp -U
[debug] Formats sorted by: hasvid, ie_pref, lang, quality, res, fps, hdr:12(7), vcodec:vp9.2(10), channels, acodec, filesize, fs_approx, tbr, vbr, abr, asr, proto, vext, aext, hasaud, source, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] ChRtyT4jZTe: Downloading 1 format(s): 0
{"id": "ChRtyT4jZTe", "formats": [{"url": "https://instagram.fdtm2-2.fna.fbcdn.net/v/t50.2886-16/299597967_841195450202344_2542793318561483375_n.mp4?_nc_ht=instagram.fdtm2-2.fna.fbcdn.net&_nc_cat=104&_nc_ohc=bFYwwMtK06gAX_oEgBI&edm=AP_V10EBAAAA&ccb=7-5&oe=6308B467&oh=00_AT_bkeQxCxvz0B0y8Ja8VoPEDYm_V8C-it9l_ENMZ2gAQQ&_nc_sid=4f375e", "width": 750, "height": 1333, "protocol": "https", "ext": "mp4", "video_ext": "mp4", "audio_ext": "none", "format_id": "0", "format": "0 - 750x1333", "resolution": "750x1333", "dynamic_range": "SDR", "http_headers": {"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, 
like Gecko) Chrome/92.0.4515.131 Safari/537.36", "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8", "Accept-Language": "en-us,en;q=0.5", "Sec-Fetch-Mode": "navigate", "Referer": "https://www.instagram.com/"}}], "title": "Video by joeel56", "description": "Monday hustle mode on \ud83d\udc69\ud83c\udffb\u200d\ud83d\udcbb\n Love the new LEDs strips on my wall \ud83e\udd70 trying to get a lot of work done this week. My current sprint is a little bit on delay \ud83c\udf1a how is your week? Btw did u saw 
my back goes \ud83e\udd97\ud83e\udd9f\ud83e\udd97 \ud83d\ude02\ud83d\ude02 \n.\n.\n.\n\n#webdeveloper #uiux #minimalist#interface #design#uxdesign#setupinspiration #programmerworld #web #userinterface #programmer #application#javascript#setup#code #minimal #website #developer #frontenddeveloper", "duration": 14.536, 
"timestamp": 1660559595, "uploader_id": "288495926", "uploader": "joeel56 Webdeveloper", "channel": "joeel56", "like_count": 2031, "comment_count": 29, "comments": [{"author": "aniakubow", "author_id": "292805263", "id": "17897864222648806", "text": "I love the LED lighting on your wall\ud83d\ude0d\ud83d\ude0d", "timestamp": 1660575194}, {"author": "alonsovallve", "author_id": "12036590578", "id": "17933159168378883", "text": "Bien \ud83d\ude4c\ud83d\ude4c\ud83d\ude4c", "timestamp": 1660592169}, {"author": "_edgeworks", "author_id": "30807455295", "id": "17922806894414967", "text": "\ud83e\udd97\ud83e\udd9f\ud83d\ude02 #setupIs 
\ud83d\udd25\ud83d\udd25", "timestamp": 1660600512}, {"author": "who.is.lyss_", "author_id": "1791382322", "id": "17955062998896842", "text": "Absolutely love your setup!!\ud83e\udd29", "timestamp": 1660687356}, 
{"author": "vanessa_flair_alexander.fikx", "author_id": "51637501075", "id": "18065043523316513", "text": 
"Hey \ud83d\udc4b\ud83c\udffb We just sent you a DM for a potential collaboration. Please reply ASAP to start. Thanks! \ud83d\ude42", "timestamp": 1660727878}, {"author": "vanessa_flair_alexander.fikx", "author_id": "51637501075", "id": "17946907283164915", "text": "Hey \ud83d\udc4b\ud83c\udffb We just sent you a DM 
for a potential collaboration. Please reply ASAP to start. Thanks! \ud83d\ude42", "timestamp": 1660727914}, {"author": "techpuni", "author_id": "54323309382", "id": "17905431374534705", "text": "That led looks so cool!", "timestamp": 1660733107}, {"author": "superintelligence_", "author_id": "30410161185", "id": "17922202127427708", "text": "Love the new lights\ud83d\ude0d", "timestamp": 1660740085}, {"author": "iblock.om", "author_id": "14785507343", "id": "18313525576061553", "text": "How much cost components computer", "timestamp": 1660797680}], "thumbnails": [{"url": "https://instagram.fdtm2-1.fna.fbcdn.net/v/t51.2885-15/299656300_768056184345908_2305085916977115048_n.jpg?stp=dst-jpg_e15_p640x640&_nc_ht=instagram.fdtm2-1.fna.fbcdn.net&_nc_cat=108&_nc_ohc=2Ii_KD0PynIAX-Tthj0&edm=AP_V10EBAAAA&ccb=7-5&oh=00_AT-g_UaiviB-625JRmczpcKb15WHp14_zflbtt6Tx3UBjA&oe=6308C9C6&_nc_sid=4f375e", "width": 640, "height": 1137, "id": "0", "resolution": "640x1137"}, {"url": "https://instagram.fdtm2-1.fna.fbcdn.net/v/t51.2885-15/299656300_768056184345908_2305085916977115048_n.jpg?stp=dst-jpg_e15&_nc_ht=instagram.fdtm2-1.fna.fbcdn.net&_nc_cat=108&_nc_ohc=2Ii_KD0PynIAX-Tthj0&edm=AP_V10EBAAAA&ccb=7-5&oh=00_AT9S1Nu0xL5hPi7hiX25CQRjpTSgUaxdt_Z6nicSK5ZMpA&oe=6308C9C6&_nc_sid=4f375e", "width": 750, "height": 1333, "id": "1", "resolution": "750x1333"}, {"url": "https://instagram.fdtm2-1.fna.fbcdn.net/v/t51.2885-15/299656300_768056184345908_2305085916977115048_n.jpg?stp=dst-jpg_e15&_nc_ht=instagram.fdtm2-1.fna.fbcdn.net&_nc_cat=108&_nc_ohc=2Ii_KD0PynIAX-Tthj0&edm=AP_V10EBAAAA&ccb=7-5&oh=00_AT9S1Nu0xL5hPi7hiX25CQRjpTSgUaxdt_Z6nicSK5ZMpA&oe=6308C9C6&_nc_sid=4f375e", "width": 1080, "height": 1920, "id": "2", "resolution": "1080x1920"}], "http_headers": {"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36", "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8", "Accept-Language": "en-us,en;q=0.5", "Sec-Fetch-Mode": "navigate", "Referer": "https://www.instagram.com/"}, "webpage_url": "https://www.instagram.com/reel/ChRtyT4jZTe/", "original_url": "https://www.instagram.com/reel/ChRtyT4jZTe/", "webpage_url_basename": "ChRtyT4jZTe", "webpage_url_domain": "instagram.com", "extractor": "Instagram", "extractor_key": "Instagram", 
"playlist": null, "playlist_index": null, "thumbnail": "https://instagram.fdtm2-1.fna.fbcdn.net/v/t51.2885-15/299656300_768056184345908_2305085916977115048_n.jpg?stp=dst-jpg_e15&_nc_ht=instagram.fdtm2-1.fna.fbcdn.net&_nc_cat=108&_nc_ohc=2Ii_KD0PynIAX-Tthj0&edm=AP_V10EBAAAA&ccb=7-5&oh=00_AT9S1Nu0xL5hPi7hiX25CQRjpTSgUaxdt_Z6nicSK5ZMpA&oe=6308C9C6&_nc_sid=4f375e", "display_id": "ChRtyT4jZTe", "fulltitle": "Video by joeel56", "duration_string": "14", "upload_date": "20220815", "requested_subtitles": null, "_has_drm": null, "requested_downloads": [{"http_headers": {"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.131 Safari/537.36", "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8", "Accept-Language": "en-us,en;q=0.5", "Sec-Fetch-Mode": "navigate", "Referer": "https://www.instagram.com/"}, "url": "https://instagram.fdtm2-2.fna.fbcdn.net/v/t50.2886-16/299597967_841195450202344_2542793318561483375_n.mp4?_nc_ht=instagram.fdtm2-2.fna.fbcdn.net&_nc_cat=104&_nc_ohc=bFYwwMtK06gAX_oEgBI&edm=AP_V10EBAAAA&ccb=7-5&oe=6308B467&oh=00_AT_bkeQxCxvz0B0y8Ja8VoPEDYm_V8C-it9l_ENMZ2gAQQ&_nc_sid=4f375e", "width": 750, "height": 1333, "protocol": "https", "ext": "mp4", "video_ext": "mp4", "audio_ext": "none", "format_id": "0", "format": "0 - 750x1333", "resolution": "750x1333", "dynamic_range": "SDR", "epoch": 1661367082, "_filename": "Video by joeel56 [ChRtyT4jZTe].mp4", "__write_download_archive": false}], "url": "https://instagram.fdtm2-2.fna.fbcdn.net/v/t50.2886-16/299597967_841195450202344_2542793318561483375_n.mp4?_nc_ht=instagram.fdtm2-2.fna.fbcdn.net&_nc_cat=104&_nc_ohc=bFYwwMtK06gAX_oEgBI&edm=AP_V10EBAAAA&ccb=7-5&oe=6308B467&oh=00_AT_bkeQxCxvz0B0y8Ja8VoPEDYm_V8C-it9l_ENMZ2gAQQ&_nc_sid=4f375e", "width": 750, "height": 1333, "protocol": "https", "ext": "mp4", "video_ext": "mp4", "audio_ext": "none", "format_id": "0", "format": "0 - 750x1333", "resolution": "750x1333", "dynamic_range": "SDR", "epoch": 1661367082, "_type": "video"}`

**Comment by bashonly** [comment]
thank you for the complete verbose log

it looks like you are running two different versions of yt-dlp:
`yt-dlp version 2022.08.14 [5593720] (pip)` with powershell 
and
`yt-dlp version 2022.08.19 [48c88e0] (win32_exe) ` from your original `-vU` log

this is why you are experiencing different behavior.

seems to me there was a regression in `2022.08.19`:
since we get `unable to extract additional data` in both logs we know that the graphql and API calls are failing in both.
 - version `2022.08.14` requests only the `/embed` version of the webpage and succeeds
 - version `2022.08.19` requests the regular webpage and then falls back to the `/embed` version of the page -- and then the `/embed` page request fails also
 
i think that requesting the regular webpage is triggering some kind of short-term rate-limit that causes the `/embed` page not to yield any useful info
 
@pritam20ps05 : we need to stop requesting the regular webpage. in my experience lately it doesn't even yield any json

**Comment by pritam20ps05** [comment]
The problem is not rate limit. The link he used was of a reel and if you check then you will find that reels embed pages gets loaded but clicking on the video won't play it. Embed for reels dosen't have any usefull info.

**Comment by bashonly** [comment]
> The problem is not rate limit. The link he used was of a reel and if you check then you will find that reels embed pages gets loaded but clicking on the video won't play it. Embed for reels dosen't have any usefull info.

okay so why do you think version `2022.08.14` (which does not request the regular webpage) works for OP while the current release does not?

**Comment by bashonly** [comment]
> The problem is not rate limit. The link he used was of a reel and if you check then you will find that reels embed pages gets loaded but clicking on the video won't play it. Embed for reels dosen't have any usefull info.

ah you are right, i was reading the log wrong. i will do some tests with your PR

**Comment by pritam20ps05** [comment]
In this case the extraction was possible because in 2022.08.14 there was a bug with embed validation due to which the extractor failed to detect a failed extraction. So his JSON metadata extraction succeeded which brought some data into media variable and due to the bug, failing the embed extraction didn't bother it, so the extractor retrived data from the JSON extraction

**Comment by thebeautyofcoding** [comment]
Thanks for working so hard. Yes, with version 2022.08.14, I am able to get all the Reels :)

**Comment by bashonly** [comment]
I think this was fixed furthermore in release 2022.10.04 (2e7675489f4323c17c8de1e1fd264365c2c36e26) and can be closed
